.github/workflows/create-release.yml

```diff
@@ -22,906 +25,794 @@ on:
-        uses: actions/upload-artifact@v4
-        with:
-          name: current-java-17-run-image-hash-code
-          path: hash-codes/build-java-17/run.oci.sha256
-
-      ## java 21 run image
-      - name: Upload java 21 run image
-        uses: actions/upload-artifact@v4
-        with:
-          name: current-java-21-run-image
-          path: ./build-java-21/run.oci
-
-      - name: Get java 21 run image hash code
-        run: |
-          mkdir -p hash-codes/build-java-21
-
-          ubi_image_current_hash_codes=($(echo '${{ needs.poll_ubi_images.outputs.ubi_image_current_hash_codes }}' | jq -r '.[]'))
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    # TODO remove before PR
+    - name: Tree
+      run: tree .
+
+    - name: Create stack ${{ matrix.stacks.name }}
+      id: create-stack
+      run: |
+        scripts/create.sh --stack-dir ${{ matrix.stacks.config_dir }} \
+                          --build-dir ${{ matrix.stacks.output_dir }}
+
+    # TODO remove before PR
+    - name: Tree
+      run: tree .
+
+    - name: Generate Package Receipts
+      id: receipts
+      run: |
+        scripts/receipts.sh --build-image "${{ matrix.stacks.output_dir }}/build.oci" \
+                            --run-image "${{ matrix.stacks.output_dir }}/run.oci" \
+                            --build-receipt current-build-receipt-${{ matrix.stacks.name }} \
+                            --run-receipt current-run-receipt-${{ matrix.stacks.name }}
+        echo "build_receipt=current-build-receipt-${{ matrix.stacks.name }}" >> "$GITHUB_OUTPUT"
+        echo "run_receipt=current-run-receipt-${{ matrix.stacks.name }}" >> "$GITHUB_OUTPUT"
+
+    # TODO remove before PR
+    - name: Tree
+      run: tree .
+
+    - name: Upload run image
+      uses: actions/upload-artifact@v4
+      with:
+        name: current-run-image-${{ matrix.stacks.name }}
+        path: "${{ matrix.stacks.output_dir }}/run.oci"
+        if-no-files-found: error 
+
+    - name: Upload build image
+      if: ${{ matrix.stacks.create_build_image == true }}
+      uses: actions/upload-artifact@v4
+      with:
+        name: current-build-image-${{ matrix.stacks.name }}
+        path: "${{ matrix.stacks.output_dir }}/build.oci"
+        if-no-files-found: error 
+
+    - name: Upload Build receipt
+      if: ${{ matrix.stacks.create_build_image == true }}
+      uses: actions/upload-artifact@v4
+      with:
+        name: current-build-receipt-${{ matrix.stacks.name }}
+        path: ${{ steps.receipts.outputs.build_receipt }}
+        if-no-files-found: error 
+
+    - name: Upload Run receipt
+      uses: actions/upload-artifact@v4
+      with:
+        name: current-run-receipt-${{ matrix.stacks.name }}
+        path: ${{ steps.receipts.outputs.run_receipt }}
+        if-no-files-found: error 
+
+  diff:
+    name: Diff Packages
+    outputs:
+      removed_with_force: ${{ steps.removed_with_force.outputs.packages_removed }}
```

Comment:
Not necessary needed in this step but wondering about these which are in the github-config version:

 build_added: ${{ steps.build_diff.outputs.added }}
      build_removed_with_force: ${{ steps.build_diff.outputs.removed }}
      run_removed_with_force: ${{ steps.run_diff.outputs.removed }}